low-level/framework/platforms/realstor/realstor_enclosure.py

```diff
@@ -38,7 +38,7 @@ class RealStorEnclosure(StorageEnclosure):
     DEFAULT_MC_IP = "127.0.0.1"
     WEBSERVICE_TIMEOUT = 20
     PERSISTENT_DATA_UPDATE_TIMEOUT = 5
-    MAX_RETRIES = 1
+    MAX_RETRIES = 2
```

Comment:
changes to this file was made in cortx-1.0 and as i heard last, Archana was planning to push to main branch also. It was never made ? This code is fine obviously and needed if not in main still!

Reply:
Yes, we need this logic in main. this logic is not in main currently.